ci_scripts/k8s-splunk.yml

```diff
@@ -77,11 +77,11 @@ spec:
         - name: SPLUNK_USER
           value: root
         - name: SPLUNK_PASSWORD
-          value: helloworld
+          value: ENV_CI_SPLUNK_PASSWORD
```

Comment:
Can you see if you can set a default value if these default env vars don't exist? It makes it easier to run these tests locally for already created methods.

Reply:
Done,
We are now matching using default values instead of parameters names which I introduced in previous commits